docs/result.md

```diff
@@ -1,14 +1,209 @@
-#Return_result
+# Return_result
 
 a POST request is invoked by the client to the server. It returns a JSON file with information like client_id, job, data, and jobs.
 
 ## JSON definition
 
 A job is communicated as a simple JSON object where key names and values are lower case:
 
-* `client_id`: a unique identifier of the client returning the result.
-* `job`: A job object, as defined in job.md. This job corresponds to the result. 
-* `data`: data collected from regulations.gov.
-	* how data is transferred has not been defined. 
-* `jobs`: list of job objects, as defined in job.md.
-	* list of new jobs identified with data. 
+* `client_id`: A unique identifier of the client returning the result.
+* `job_id`: ID of the current job object.
+* `data`: A path describing where the raw JSON file recieved from querying regulations, as well as the raw JSON data.
+* `jobs`: A list of new job objects. These job objects omit the job_id field. Jobs are retrieved through the JSON data collected from the regulations server.
+	* Documents have a list of docket and document jobs
+  * Document can have a list of download jobs
+  * Docket and downloads will not return any new jobs
+
+## Data Storage
+
+```
+Data (folder)
+  Agency (folder)
+    DocketID (folder)
+      docket.json
+      DocumentID (folder)
+        basic_document.json
+        document.json
+```     
+
+## Example
+
+* Docket
+```
+{
+  'client_id': '123ABC',
+  'job_id': 'ABC123',
```

Comment:
AS I pointed out in class, my choice of ABC123 and 123ABC was poor.  Let's change them to `client15` and `job42`.  This will help when we want to discuss them in meetings or in presentations.